e2e/cypress/tests/integration/system_console/environment_spec.js

```diff
@@ -168,10 +166,9 @@ describe('Environment', () => {
                 const url = imageDiv.css('background-image').split('"')[1];
 
                 // # Verify that the image is the correct one
-                cy.fixture(mattermostIcon).then((overrideImage) => {
+                cy.fixture(mattermostIcon).then(() => {
                     cy.request({url, encoding: 'base64'}).then((response) => {
                         expect(response.status).to.equal(200);
-                        expect(response.body).to.eq(overrideImage);
```

Comment:
dont we want to verify the image here anymore?

Reply:
oh i see i am ok with leaving it out then.

Reply:
may be just check if body is not empty then.

Reply:
will add this